packages/vuetify/src/components/VColorPicker/VColorPickerPreview.tsx

```diff
@@ -3,18 +3,25 @@ import './VColorPickerPreview.sass'
 
 // Components
 import { VSlider } from '@/components/VSlider'
+import { VIcon } from '@/components/VIcon'
 
 // Composables
 import { makeComponentProps } from '@/composables/component'
 
 // Utilities
-import { defineComponent, HSVtoCSS, propsFactory, useRender } from '@/util'
+import { defineComponent, HexToHSV, HSVtoCSS, propsFactory, useRender } from '@/util'
 import { nullColor } from './util'
 
 // Types
 import type { PropType } from 'vue'
 import type { HSV } from '@/util'
 
+declare global {
+  interface Window {
+      EyeDropper?: any;
+  }
+}
```

Comment:
Move to globals.d.ts and maybe add a proper interface instead of any

Reply:
Moved to global.d.ts